### PR TITLE
FE libs Tailwind - part 5

### PR DESCRIPTION
**Pull request by goranalkovic-infinum**
# Description

- adds first round of docs for FE libs Tailwind
- updated demo theme

# Screenshots / Videos

\-

# Linked documentation PR

\-

### CODE REVIEW

File: website/docs/tailwind/helpers.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+	// highlight-next-line
+	const hiddenOptions = getHiddenOptions(hideOptions);
+
+	...
+
+	return (
+		<>
+			<Toggle
+				label={__('my option', 'fe-libs-tailwind')}
```
Comment (dadadavorin) [inline website/docs/tailwind/helpers.md:36]:
```suggestion
				label={__('My option', 'fe-libs-tailwind')}
```

**Review by dadadavorin — APPROVED** [review]
👌

**Review by dadadavorin — DISMISSED** [review]

File: website/docs/tailwind/helpers.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+### `getHiddenOptions`
+In order to simplify hiding parts of component options, the `getHiddenOptions` helper was created.
+
+To use, first in your component add:
+```jsx
+export const DemoOptions = (attributes) => {
+	const {
+		setAttributes,
+		// highlight-next-line
```
Comment (piqusy) [inline website/docs/tailwind/helpers.md:20]: Curious to se how this looks in action... can you please provide a live example

File: website/docs/tailwind/helpers.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+	// highlight-next-line
+	const hiddenOptions = getHiddenOptions(hideOptions);
+
+	...
+
+	return (
+		<>
+			<Toggle
+				label={__('My option', 'fe-libs-tailwind')}
```
Comment (piqusy) [inline website/docs/tailwind/helpers.md:36]:
```suggestion
				label={__('My option', 'fe-libs-tailwind')}
```
Text domain has changed from `eightshift-frontend-libs`? I understand the `tailwind` addition but the change is significant here, can we have it somewhat similar... maybe I'm missing something here 😄

**Review by piqusy — CHANGES_REQUESTED** [review]
Nice work 👏🏼 

Left a suggestions / question

File: website/docs/tailwind/helpers.md (hunk 2 above)
Comment (goranalkovic-infinum) [reply website/docs/tailwind/helpers.md:20]: Playground 😉

File: website/docs/tailwind/helpers.md (hunk 3 above)
Comment (goranalkovic-infinum) [reply website/docs/tailwind/helpers.md:36]: This is a sample component, the textdomain will change based on your theme

**Review by iobrado — DISMISSED** [review]

**Review by iobrado — APPROVED** [review]

File: website/docs/tailwind/helpers.md (hunk 2 above)
Comment (piqusy) [reply website/docs/tailwind/helpers.md:20]: Found it, not in the playground tho' 😄 
image

Comment (piqusy) [reply website/docs/tailwind/helpers.md:20]: Not sure what you have in mind with the "interactive"

**Review by piqusy — DISMISSED** [review]

**Review by iobrado — APPROVED** [review]

**Review by dadadavorin — APPROVED** [review]

File: website/docs/tailwind/helpers.md (hunk 2 above)
Comment (goranalkovic-infinum) [reply website/docs/tailwind/helpers.md:20]: Ah, I thought you meant you want an interactive example hahah

**Pull request merged**